docs/zenith/index.md

```diff
@@ -0,0 +1,54 @@
+
+Dagger is an integrated platform to orchestrate the delivery of applications to the cloud from start to finish. The Dagger Platform includes the Dagger Engine, Dagger Cloud, and the Dagger SDKs.
+
+### What is the Dagger Engine?
+
+The Dagger Engine is a programmable tool that lets you replace your software project's artisanal scripts with a modern API and cross-language scripting engine.
+
+1. Encapsulate all your project's tasks and workflows into simple functions, written in your programming language of choice.
+2. Dagger packages your functions into a custom GraphQL API.
+3. Run your functions from the command line, your language interpreter, or a custom HTTP client.
+4. Package your functions into a module, to reuse in your next project or share with the community.
+5. Search the [Daggerverse](https://daggerverse.dev) for existing modules, and import them into yours. All Dagger modules can reuse each other's functions - regardless of which language they are written in.
+
+### What is Dagger Cloud?
+
+Dagger Cloud complements the Dagger Engine with a production-grade control plane. Features of Dagger Cloud include pipeline visualization, operational insights, and distributed caching.
+
+## Who is it for?
+
+Dagger may be a good fit if you are...
+
+* Your team's "designated devops person", hoping to replace a pile of artisanal scripts with something more powerful.
+* A platform engineer writing custom tooling, with the goal of unifying application delivery across organizational silos.
+* A cloud-native developer advocate or solutions engineer, looking to demonstrate a complex integration on short notice.
+
+### Benefits to development teams
+
+* Reduce complexity: Even complex builds can be expressed as a few simple functions.
+* No more "push and pray": Everything CI can do, your local dev environment can do too.
+* Native language benefits: Use the same programming language to develop your application and its delivery tooling.
+* Easy onboarding of new developers: If you can build, test and deploy - they can too.
+* Caching by default: Dagger caches everything. Expect 2x to 10x speedups.
+* Cross-team collaboration: Reuse another team's workflows without learning their stack.
+
+### Benefits to platform teams
+
+* Reduce CI lock-in: Dagger functions run on all major CI platforms - no proprietary DSL needed.
+* Eliminate bottlenecks: Let application teams write their own functions. Enable standardization by providing them a library of reusable components.
+* Save time and money with faster CI runs: CI pipelines that are "Daggerized" typically run 2x to 10x faster, thanks to caching and concurrency. This means developers waste less time waiting for CI, and you spend less money on CI compute.
+* Benefit from a viable platform strategy: Development teams need flexibility, and you need control. Dagger gives you a way to reconcile the two, in an incremental way that leverages the stack you already have.
+
+## How does it work?
+
+TODO
```

Comment:
@sipsma can you please add some info here, using https://docs.dagger.io/#how-does-it-work as a model?

Reply:
Yep, there's definitely a few tweaks to make for modules and the fact that we're CLI-centric (rather than SDK-centric) in terms of what executes on the host now.

Will push an update tomorrow

Reply:
@vikram-dagger Pushed an update for this. Not super happy w/ how the mermaid diagram renders (at least when I previewed it on github), but let me know what you think overall or any fixes we could make to improve its appearance.

Reply:
The diagram looked extra bad in the preview deployment of the site so I pushed another improvement, which required that I removed `OCI runtime` unfortunately. Otherwise it was weirdly entangled:


<img width="627" alt="Screenshot 2023-11-02 at 1 03 05 AM" src="https://github.com/dagger/dagger/assets/30126853/34da2d30-e653-45f3-8c00-706fb661e00b">

Unfortunately the diagram is still cutting names off after the update:
<img width="763" alt="Screenshot 2023-11-02 at 1 09 59 AM" src="https://github.com/dagger/dagger/assets/30126853/d94513c5-4f45-4188-be46-e8f06ac9bdc2">

Not sure if I just need to shorten names even more or if there's other formatting options available I'm not aware of.

Reply:
Are you seeing the names being shortened in GitHub, or in the deploy preview? I just checked the deploy preview and it looks fine.

Reply:
@sipsma two comments:
- In the diagram and explanation, I'm not able to grok the relationship between the modules and the build/test/etc pipelines.
- In the explanation, I was confused by usage of "API" vs "function". Also, in some of the steps, I couldn't understand whether it was referring to the core API or the module API. I have made a few inline suggestions in an attempt to add greater precision, however please review as I may have misunderstood.